### PR TITLE
Adds "wq" command as a combination of "write" and "quit".

### PR DESCRIPTION
**Pull request by mucinoab**
As a heavy Vim user I use this command **a lot** and since the release of Dijo the absence of it has bothered me (because muscle memory). 

I think the usual behavior of a Dijo user (or at least my own) is as follows:
- Complete the task you are tracking
- Open Dijo and mark the task as done
- Save your progress with _w_ or _write_ and press enter
- Quit with _q_ or _quit_ and press enter 

So it would be much nicer if we could just do _wq_ and enter.

This was a quick implementation so please let me know of any problems you may encounter.

### CODE REVIEW

File: src/app/impl_self.rs [hunk 1]
```diff
@@ -260,6 +261,7 @@ impl App {
                         self.message.set_message("help <command>|commands|keys")
                     }
                 }
+                Command::WriteAndQuit => self.save_state(),
```
Comment (NerdyPepper) [inline src/app/impl_self.rs:264]: could this match arm be merged with the one below it?

File: src/command.rs [hunk 2]
```diff
@@ -196,6 +199,7 @@ impl Command {
             }
             "mprev" | "month-prev" => return Ok(Command::MonthPrev),
             "mnext" | "month-next" => return Ok(Command::MonthNext),
+            "wq" | "writeandquit" => return Ok(Command::WriteAndQuit),
```
Comment (NerdyPepper) [inline src/command.rs:202]: i think we can remove the `writeandquit` alias, after all, vim's `wq` is a composition of two commands.

**Comment by NerdyPepper** [comment]
Sweet! It did occur to me that `wq` would be more ergonomic, thanks for going ahead and implementing it!

I have requested a couple of changes, let me know what you think.

**Comment by mucinoab** [comment]
@NerdyPepper Done, thanks for the quick replay, I am glad you liked the changes.

**Comment by NerdyPepper** [comment]
Looks good, thanks for the contribution!

**Pull request merged**